source/features/archive-forks-link.tsx

```diff
@@ -16,7 +16,7 @@ function addLinkToBanner(banner: HTMLElement): void {
 }
 
 function init(signal: AbortSignal): void {
-	observe('#js-repo-pjax-container > .flash-warn:first-child', addLinkToBanner, {signal});
+	observe('#js-repo-pjax-container > .flash-warn', addLinkToBanner, {signal});
```

Comment:
The problem is that sometimes there's more than one banner I think. Can you `git blame` the line to find out when that happens? Maybe `:first-of-type` still works?

Reply:
`:first-of-type` works